config/checkstyle/suppressions.xml

```diff
@@ -10,10 +10,9 @@
     <suppress checks="Javadoc.*" files=".*Test.java"/>
     <suppress checks="MethodName" files=".*Logger.java"/>
     <suppress checks="MethodName" files=".*LoggerTest.java"/>
-    <suppress checks="MemberName" files="AuthenticationContext.java" lines="813"/>
-    <suppress checks="VisibilityModifier" files="AuthenticationContext.java" lines="813"/>
     <suppress checks="MemberName" files="SSOStateSerializer.java" lines="47"/>
     <suppress checks="HideUtilityClassConstructor" files="ReflectionUtils.java" lines="31"/>
```

Comment:
will do after heidi merges her pr

---
In reply to: [76142647](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76142647) [](ancestors = 76142647)